buildscripts/Build-NetCore.ps1

```diff
@@ -1,40 +1,39 @@
-$frameworkVersion = "1.0.0-rc1-update1"
-$frameworkArchitecture = "x86"
+$dotnetHome = Join-Path (Get-Location) ".dotnet"
+$dotnetPath = Join-Path $dotnetHome "bin"
 
-$dnxHome = Join-Path (Get-Location) ".dnx"
-$dnxPath = Join-Path $dnxHome "bin"
+Write-Host "Downloading install.ps1 to: $dotnetPath"
+if (!(Test-Path $dotnetPath)) { md $dotnetPath | Out-Null }
 
-Write-Host "Downloading dnvm to: $dnxPath"
-if (!(Test-Path $dnxPath)) { md $dnxPath | Out-Null }
-
-$dnvmPs1Path = Join-Path $dnxPath "dnvm.ps1"
-$dnvmCmdPath = Join-Path $dnxPath "dnvm.cmd"
+$installPs1Path = Join-Path $dotnetPath "install.ps1"
 
 $wc = New-Object System.Net.WebClient
-$wc.DownloadFile('https://raw.githubusercontent.com/aspnet/Home/dev/dnvm.ps1', $dnvmPs1Path)
-$wc.DownloadFile('https://raw.githubusercontent.com/aspnet/Home/dev/dnvm.cmd', $dnvmCmdPath)
+$wc.DownloadFile('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/install.ps1', $installPs1Path)
+
+$env:InstallDir = $dotnetHome
+if (!(Test-Path env:\DOTNET_CLI_CHANNEL)) { $env:DOTNET_CLI_CHANNEL = "beta" }
+if (!(Test-Path env:\DOTNET_CLI_VERSION)) { $env:DOTNET_CLI_VERSION = "1.0.0-preview1-002702" }
 
-Write-Host "Downloading dnx"
+Write-Host "Downloading dotnet/cli"
 
-$env:PATH = ("$dnxPath;" + $env:PATH)
+. $installPs1Path -Channel $env:DOTNET_CLI_CHANNEL -InstallDir $dotnetHome -Version $env:DOTNET_CLI_VERSION
 
-dnvm install $frameworkVersion -runtime coreclr -arch $frameworkArchitecture
+$env:PATH = ("$dotnetPath;" + $env:PATH)
 
-dnx --version
+dotnet --info
 
 Write-Host "Downloading packages"
 
-dnu restore
+cd src
+dotnet restore -v Minimal --disable-parallel
```

Comment:
We haven't seen issues with parallel restores in any of our repos. Is there a specific reason you're disabling it?